servicetalk-http-api/src/main/java/io/servicetalk/http/api/HttpProtocolVersion.java

```diff
@@ -149,6 +149,24 @@ public String name() {
         return httpVersion;
     }
 
+    /**
+     * Resolves and return the http version number as a String. e.g. : 1.1 or 2.0.
+     * @return the http version number string.
+     */
+    public String fullVersion() {
+        if (major == 1) {
```

Comment:
Instead of special casing, consider caching `major + '.' + minor` in a local variable in the constructor and then just returning that `fullVersion` string directly.

Reply:
Good idea. Storing that instead.